databricks/koalas/plot.py

```diff
@@ -673,7 +694,41 @@ def kde(self, bw_method=None, ind=None, **kwds):
     density = kde
 
     def area(self, **kwds):
-        return _unsupported_function(class_name='pd.Series', method_name='area')()
+        """
+        Draw a stacked area plot.
+
+        An area plot displays quantitative data visually.
+        This function wraps the matplotlib area function.
+
+        Parameters
+        ----------
+        x : label or position, optional
+            Coordinates for the X axis. By default uses the index.
+        y : label or position, optional
+            Column to plot. By default uses all columns.
+        stacked : bool, default True
+            Area plots are stacked by default. Set to False to create a
+            unstacked plot.
+        **kwds : optional
+            Additional keyword arguments are documented in
+            :meth:`DataFrame.plot`.
+
+        Returns
+        -------
+        matplotlib.axes.Axes or numpy.ndarray
+            Area plot, or array of area plots if subplots is True.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({
+        ...     'sales': [3, 2, 3, 9, 10, 6],
+        ...     'signups': [5, 5, 6, 12, 14, 13],
+        ...     'visits': [20, 42, 28, 62, 81, 50],
+        ... }, index=pd.date_range(start='2018/01/01', end='2018/07/01',
+        ...                        freq='M'))
```

Comment:
Seems like there are some bugs in lower pandas versions. I can't use existing test way in `test_series_plot.py`. It passes in my local but fails in Travis in certain condition (failed from pandas library).